build.gradle

```diff
@@ -86,34 +86,15 @@ task ci(type: WriteProperties) {
     property 'tagName', tagName
 }
 
-jib {
-    from {
-        image = 'eclipse-temurin:17-jdk-focal'
-    }
-    to {
-        image = imageName
-        tags = [tagName.toString()]
-    }
-    container {
-        mainClass = mainClassName
-        ports = ['8080']
-    }
-}
-
-def ssrImageName = 'docker.io/molgenis/ssr-catalogue'
-def ssrTagName = project.version.toString()
-if (version.toString().endsWith('-SNAPSHOT')) {
-    ext.hash = 'git rev-parse --short HEAD'.execute().text.trim()
-    ssrImageName = "docker.io/molgenis/ssr-catalogue-snapshot"
-    ssrTagName = "${project.version.toString()}-${ext.hash}"
-}
-
 docker {
-    name ssrImageName
-    tags 'latest', ssrTagName
-    dockerfile file('Dockerfile')
-    files 'apps'
```

Comment:
outdates , done by jenkins job

Reply:
yes and no, I don't like the jenkins job because it ties us to jenkins and makes it harder to switch fast to another build system or run it locally if needed (e.g. we will soon  move to circle-ci). Hence I have moved this logic to build.gradle see above.

Reply:
Alternatively we could make seperate bunch of scripts, see how this PR is developing #2324